### PR TITLE
themes fixPack

### PR DESCRIPTION
**Pull request by vik753**
…xed selected year/month bg color, [RangeDatePicker]: fixed error border color, [Slider]: fixed tooltip & caption colors, [Switch]: fixed toggler and border colors, [ProgressBar]: fixed border-radius, [StatusIndicator]: fixed 24size font-weight.

### CODE REVIEW

File: uui/components/inputs/Switch.module.scss [hunk 1]
```diff
@@ -36,14 +36,14 @@
...
-    --uui-switch-border-hover: var(--uui-neutral-40);
+    --uui-switch-border-hover: var(--uui-control-bg-hover);
     --uui-switch-border-checked: var(--uui-color-50);
     --uui-switch-border-disabled: var(--uui-control-border-disabled);
     --uui-switch-border-checked-hover: var(--uui-color-60);
     --uui-switch-border-checked-disabled: var(--uui-color-20);
     --uui-switch-label: var(--uui-control-label);
     --uui-switch-label-disabled: var(--uui-control-label-disabled);
-    --uui-switch-toggler-bg: var(--uui-surface-main);
+    --uui-switch-toggler-bg: var(--uui-neutral-0);
```
Comment (AlekseyManetov) [inline uui/components/inputs/Switch.module.scss:46]: can we use --uui-control-bg here?

Comment (vik753) [reply uui/components/inputs/Switch.module.scss:46]: We need #FFFFFF color in all themes by design, but --uui-control-bg =--uui-neutral-70 in DARK theme, so we can't.

File: uui/components/datePickers/RangeDatePicker.module.scss [hunk 2]
```diff
@@ -63,7 +63,7 @@
         }
     }
 
-    &:global(.uui-focus) {
+    &:global(.uui-focus):not(:global(.uui-invalid)) {
```
Comment (siarhei-epam) [inline uui/components/datePickers/RangeDatePicker.module.scss:66]: I'm not sure if that's right, I think we should still show focus.

Comment (vik753) [reply uui/components/datePickers/RangeDatePicker.module.scss]: restored focus

**Review by siarhei-epam — APPROVED** [review]

**Pull request merged**